docs/plugins/preset-stage-1.md

```diff
@@ -8,13 +8,14 @@ package: babel-preset-stage-1
 
 This preset includes the following plugins:
 
-- [transform-class-constructor-call](/docs/plugins/transform-class-constructor-call) (Deprecated)
-- [transform-export-extensions](/docs/plugins/transform-export-extensions)
+- [transform-class-constructor-call](/docs/plugins/transform-class-constructor-call/) (Deprecated)
+- [transform-export-extensions](/docs/plugins/transform-export-extensions/)
+- [transform-unicode-property-regex](/docs/plugins/transform-unicode-property-regex/)
```

Comment:
These links should all have a trailing slash to prevent useless HTTP redirects.

```sh
$ curl -sI 'https://babeljs.io/docs/plugins/transform-export-extensions'
HTTP/1.1 301 Moved Permanently
[…]
Location: http://babeljs.io/docs/plugins/transform-export-extensions/
[…]
```